README.md

```diff
@@ -1,3 +1,57 @@
+## This is a fork of KoboldAI that implements 4bit GPTQ quantized support to include Llama.
+
+### Install/Use Guide
+(This guide is for both Linux and Windows and assumes user has git installed and a basic grasp of command line use)
+
+In the command prompt/command line navigate to where you want the KoboldAI subfolder to be created.
+
+git clone https://github.com/0cc4m/KoboldAI -b latestgptq --recurse-submodules
+
+cd KoboldAI
+
+Next step, subfolder mode or B: option doesn't matter choose either
+
+[if on Windows] install_requirements.bat if it closes the window when it finishes, reopen a command prompt and navigate back to your KoboldAI directory.
+
+[if on Linux] install_requirements.sh
+
+
+[if on Windows] run commandline.bat
+
+[if on Linux] run commandline.sh
+
+commandline.bat/commandline.sh will put you in KoboldAI's virtual environment (as shown by (base) affixed to the prompt).
+
+
+cd repos
+
+cd gptq
+
+
+[if on Windows, Visual Studio 2019 must be installed with C++ compiler option] python setup_cuda.py install
+
+[if on Linux] python setup_cuda.py install
```

Comment:
This step required cuda version 11.3 to complied. But cuda 11.3 did not have support for Ubuntu 22.04, anyway to circumvent this?